controllers/front/CartController.php

```diff
@@ -621,8 +657,12 @@ protected function areProductsAvailable()
 
             if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
                 return $this->trans(
-                   'The item %product% in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
-                    ['%product%' => $product['name']],
+                    'The product [1]%product%[/1] is now a product with combinations. Please select one to proceed with your order.',
+                    [
+                        '%product%' => $product['name'] . (isset($product['attributes']) && !empty($product['attributes']) ? ' (' . $product['attributes'] . ')' : ''),
```

Comment:
```suggestion
                        '%product%' => $product['name'] . ($product['attributes'] ? ' (' . $product['attributes'] . ')' : ''),
```
`attributes` are always in the array. Right? It's `'attributes' => ''` when product doesn't have attributes. Besides there is already a check `$currentProduct->hasAttributes()` above.

Is anyone able to trigger this alert? Adding simple product to the cart and then making it a product with combinations doesn't trigger it. So what does?